### PR TITLE
add ArticleRatings for T848

### PR DESCRIPTION
**Pull request by ImBoPhil**

### CODE REVIEW

**Comment by LegoFan4000** [comment]
Again please add  it to the media wiki repo first.

**Comment by ImBoPhil** [comment]
I did do the Mediawiki repo

File: LocalExtensions.php [hunk 1]
```diff
@@ -24,6 +24,10 @@
 	wfLoadSkin( 'apex' );
 }
 
+if ( $wmgUseArticleRatings ) {
+	require_once( "$IP/extensions/ArticleRatings/ArticleRatings.php" );
```
Comment (LegoFan4000) [inline LocalExtensions.php]: wfLoad method is required.

**Comment by Reception123** [comment]
@LegoFan4000 The pull request is correct. JohnLewis suggested to use require_once generally, as wfload doesn't work with some extensions

**Pull request merged**